py/desisim/scripts/quickquasars.py

```diff
@@ -299,6 +301,11 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
             log.info("Creating dir {}".format(pixdir))
             os.makedirs(pixdir)
 
+    if not eboss is None:
+        dwave_out=1.   #to keep the same value used for DR16
```

Comment:
@alxogm - I'm worried that eBOSS users might think that dwave_out will also affect them, and here we hide its effect. We could rename the adgument to dwave_DESI, or at least edit the help string to make it clear it only affects DESI. May be add also a print warning here if you were trying to run with eBOSS and also trying to change the value of dwave_out?

Reply:
Actually, the eBOSS version should really just ignore dwave_out, since the SDSS spectrograph doesn't have constant pixels in wavelength, but logarithmic. 

I'd suggest to set dwave_out=None when running with eBOSS flag.

Reply:
Hi @andreufont, right! 

dwave_out=None was my first option, but I changed to 1 because I was not considering the there is an eBOSS branch that is not yet merged, so I didn't see what you mention about eBOSS not having constant pixels. So I've set it back to dwave_out=None, and changed the argument name also, to avoid confusion...